code/controllers/Processes/air.dm

```diff
@@ -107,6 +127,7 @@ var/global/datum/controller/process/air_system/air_master
 			return
```

Comment:
correct me if I'm wrong, but if it does this, won't it return and never scheck()?